### DISCUSSION TITLE
Add Get Started page

### DISCUSSION THREAD

**Discussion by pgrandinetti** [scalar/scalar#957] [Q&A]
Hi. Can you suggest ideas about how to add, and include in the side Nav, a page that's like a user guide, or tutorial, and therefore not in yaml format for open api?

**Comment by hanspagel** [comment]
Great idea! It’s on my list to set up a docs page (or to at least add more content to the README). ⭐

**Reply by pgrandinetti** [reply]
@hanspagel  Can you give me some hints about how to do it?
